<?xml version="1.0" encoding="utf-8"?>
<launch>
	<!-- -->
	<node name="gazePublisher" pkg="myPupilLab" type="gazePub.py" output="screen" args=""></node>
	<node name="worldImagePublisher" pkg="myPupilLab" type="worldCameraPub.py" output="screen" args=""></node>
	<node name="vocus2" pkg="vocus2_ros" type="vocus2_ros_node" output="screen"></node>

</launch>
